### PR TITLE
Add ignore hash check flag when downloading

### PR DESCRIPTION
**Pull request by SteveNewson**
- [x] All [tests](https://github.com/jfrog/jfrog-cli-core#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli-core/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----

The core client code has been updated to include a flag which can ignore
hash check failures during downloads:

https://github.com/jfrog/jfrog-client-go/pull/629

This change passes the flag value through to the client code. The CLI
client will still need to be enhanced to add a command line option which
sets this flag. 

This relates to the feature request: https://github.com/jfrog/jfrog-cli/issues/1636

### CODE REVIEW

**Comment by SteveNewson** [comment]
@yahavi would you be able to trigger frogbot to start this build please? This follows on from the changes I made to the go client library.

File: go.mod [hunk 1]
```diff
@@ -13,7 +13,7 @@ require (
 	github.com/jedib0t/go-pretty/v6 v6.3.5
 	github.com/jfrog/build-info-go v1.4.0
 	github.com/jfrog/gofrog v1.2.0
-	github.com/jfrog/jfrog-client-go v1.19.0
+	github.com/jfrog/jfrog-client-go v1.19.1-0.20220810150424-4126c4b5aea7
```
Comment (yahavi) [inline go.mod:16]: Please uncomment the "replace" at the bottom of the file and add the version there
```suggestion
	github.com/jfrog/jfrog-client-go v1.19.0
```

**Review by yahavi — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (SteveNewson) [reply go.mod:16]: Gotcha!

**Comment by yahavi** [comment]
@SteveNewson,
Thanks a lot for your time and for this feature. We really appreciate it!
We had a team discussion about it. We decided it would be better to name the flag "SkipChecksum" instead of "IgnoreHashCheck". 
Could you please consider renaming it accordingly?

**Review by yahavi — APPROVED** [review]
Thanks for this contribution, @SteveNewson!

**Pull request merged**